src/components/Block/Block.styles.ts

```diff
@@ -17,13 +18,15 @@ const getMarginTop = (topSpacing?: GutterSize, push?: number) => {
 
 export const getStyles = (props: BlockProps & { withinClickable: boolean }): IRawStyle => {
   const { push, textSize, topSpacing, bottomSpacing, textAlign, textColor, withinClickable } = props;
+  const theme = getTheme();
+  const font = textSize ? theme.fonts[textSize === TextSize.MEDIUM_SUB ? 'smallPlus' : textSize] : undefined;
```

Comment:
would this match Fabric's naming? the only issue is that this would be a breaking change, so we'd need to version-bump YamJS and MC and fix all instances